### ISSUE TITLE
Custom namespace triggers false positive error for parameters passed by rvalue reference

### ISSUE THREAD

**Issue by johnthagen** [theandrewdavis/cpplint#10]
Consider the following code in `test.cpp`:

```cpp
// Copyright [year] <Copyright Owner>

namespace name {
    using Color = int;
}

class Class {
    void method(name::Color&& val);
};

void Class::method(name::Color&& val) {
    return;
}

```

```bash
$ pip list
...
cpplint (1.2.1)
$ cpplint --filter=-build/c++11 test.cpp
test.cpp:8:  Missing spaces around &&  [whitespace/operators] [3]
test.cpp:11:  Missing spaces around &&  [whitespace/operators] [3]
Done processing test.cpp
Total errors found: 2
```

If you remove the custom namespace `name::` from the parameter types, `cpplint` does not throw errors, so that seems to be tripping up the analysis.
